payments/swish/seamless-view.md

```diff
@@ -18,120 +18,389 @@ sidebar:
+   Pay will check if the card is enrolled with 3-D Secure. This depends on the
+   issuer of the card. If the card is not enrolled with 3-D Secure, no
+   authentication of the cardholder is done.
+2. Card supports 3-D Secure - if the card is enrolled with 3-D Secure, Swedbank
+   Pay will redirect the cardholder to the autentication mechanism that is
+   decided by the issuing bank. Normally this will be done using BankID or
+   Mobile BankID.
 
-All valid options when posting in a payment with operation equal to Purchase,
-are described in [the technical reference][swish-payments].
+### Payment Url
 
-#### General
+{% include payment-url.md
+when="at the 3-D Secure verification for Card Payments" %}
 
-* **Defining CallbackURL**: When implementing a scenario, it is optional to set
-  a [CallbackURL][callback-url] in the `POST` request. If callbackURL is set
-  Swedbank Pay will send a postback request to this URL when the consumer has
-  fulfilled the payment.
+## Seamless View Back End
 
-## Purchase flow
+When properly set up in your merchant/webshop site and the payer starts the
+purchase process, you need to make a POST request towards Swedbank Pay with your
+Sales information. This will generate a payment object with a unique
+`paymentID`. You will receive a **JavaScript source** in response.
 
-The sequence diagram below shows the three requests you have to send to Swedbank
-Pay to make a purchase. The links will take you directly to the API description
-for the specific request.
+### Intent
 
-```mermaid
-sequenceDiagram
-  activate Browser
-  Browser->>-Merchant: start purchase
-  activate Merchant
-  Merchant->>-SwedbankPay: POST <Create Swish payment> (operation=PURCHASE)
-  activate SwedbankPay
-  note left of Merchant: First API request
-  SwedbankPay-->>-Merchant: payment resource
-  activate Merchant
-
-  Merchant-->>-SwedbankPay: POST <Create Sales Transaction> (operation=create-sale)
-  activate SwedbankPay
-  SwedbankPay-->>-Merchant: sales resource
-  activate Merchant
-
-  note left of Merchant: POST containing MSISDN
-  Merchant--x-Browser: Tell consumer to open Swish app
-
-  Swish_API->>Swish_App: Ask for payment confirmation
-  activate Swish_App
-  Swish_App-->>-Swish_API: Consumer confirms payment
-  activate Swish_API
-
-  Swish_API-->>-SwedbankPay: Payment status
-  activate SwedbankPay
-  SwedbankPay-->>-Swish_API: Callback response
-  activate Swish_API
-  Swish_API->>-Swish_App: Start redirect
-  activate Swish_App
-
-  Swish_App--x-Browser: Redirect
-  Merchant->>SwedbankPay: GET <Sales transaction>
-  activate SwedbankPay
-  SwedbankPay-->>-Merchant: Payment response
-  activate Merchant
-  Merchant-->>-Browser: Payment Status
+{% include intent.md %}
+
+### Operations
+
+The API requests are displayed in the purchase flow above.
+You can [create a card `payment`][create-payment] with following `operation`
+options:
+
+* [Purchase][purchase] (We use this value in our examples)
```

Comment:
A bullet list containing only one element seems redundant. Perhaps rewrite the top section to show there are no options?

Reply:
Redundant indeed. Better now!